modules/passkey/hardhat.config.ts

```diff
@@ -1,15 +1,40 @@
 import '@nomicfoundation/hardhat-toolbox'
+import dotenv from 'dotenv'
 import type { HardhatUserConfig } from 'hardhat/config'
 import 'hardhat-deploy'
 
+// Load environment variables.
+dotenv.config()
+
 const config: HardhatUserConfig = {
   paths: {
     artifacts: 'build/artifacts',
     cache: 'build/cache',
     deploy: 'src/deploy',
     sources: 'contracts',
   },
-  solidity: '0.8.24',
+  networks: {
+    localhost: {
+      url: 'http://localhost:8545',
+      tags: ['dev', 'safe'],
+    },
+    hardhat: {
+      blockGasLimit: 100000000,
+      gas: 100000000,
+      gasPrice: 10000000000,
+      tags: ['test'],
+    },
+  },
+  solidity: {
+    compilers: [
+      {
+        version: '0.8.24',
+      },
+      {
+        version: '0.8.23',
```

Comment:
We can just use 0.8.23 everywhere to simplify the setup as well.

Reply:
Maybe it makes sense to include the Safe 4337 module only as an artifact instead of building it for the tests?

Reply:
I'm fine as-is though.

Reply:
Forgot it was an option :facepalm: done

Reply:
> Maybe it makes sense to include the Safe 4337 module only as an artifact instead of building it for the tests?

So TIL that if you use artifacts directly that will not produce hardhat artifacts and you won't be able to use ethers.getContractAt(name) to get the contract. https://github.com/safe-global/safe-modules/pull/306/commits/c7c79a668e24cef46c250a19900cf761d01b1a11